test-cases/artifacts/ubuntu2004-fips.yaml

```diff
@@ -0,0 +1,21 @@
+ami_id_db_scylla: 'ubuntu-pro-fips-server/images/hvm-ssd/ubuntu-focal-20.04-amd64-pro-fips-server-20221121-7bc828d1-c072-4d33-a989-fbad50380cfb'  # 'ami-03cf7ddd346310b5f'
+root_disk_size_db: 50
+backtrace_decoding: false
+cluster_backend: 'aws'
+instance_type_db: 'i3.large'
+instance_provision: "spot"
+instance_provision_fallback_on_demand: true
+logs_transport: 'ssh'
+n_db_nodes: 1
+n_loaders: 0
+n_monitor_nodes: 0
+nemesis_class_name: 'NoOpMonkey'
+region_name: 'us-east-1'
+scylla_linux_distro: 'ubuntu-focal'
+scylla_repo: 'http://downloads.scylladb.com/unstable/scylla/master/deb/unified/latest/scylladb-master/scylla.list'
+use_preinstalled_scylla: false
+ami_db_scylla_user: 'ubuntu'
+client_encrypt: true
```

Comment:
And this work? I'm quite surprised it would without some work on c-s calls

Reply:
Actually we don't have clients in this case, so I'm not sure how valuable it is.

Reply:
> And this work? I'm quite surprised it would without some work on c-s calls

No no, hence (partially) the Draft. The c-s fails

> Actually we don't have clients in this case, so I'm not sure how valuable it is.

We actually do have 2 stresses that are executed 3 times: https://github.com/scylladb/scylla-cluster-tests/blob/master/artifacts_test.py#L123

Reply:
And cqlsh calls, and few calls connecting with python-driver.

So yeah we do have clients

Reply:
> And cqlsh calls, and few calls connecting with python-driver.
> 
> So yeah we do have clients

I updated the stresses and now they work, but I didn't change anything regarding any "cqlsh calls". Is it possible that it happens automatically?